.gitlab-ci.yml

```diff
@@ -98,15 +96,11 @@ check:format:
   stage: test
 
 test:
-  services:
-    - mongo:4.2.6
   script:
-    # variables defined alongside services are prepended
-    # with the service name, to test the service we need to
-    # set the variable inline
-    - MONGODB_CONNECTION_STRING=mongodb://mongo:27017/asap
-    - yarn workspace @asap-hub/users-service migrate up
     - yarn run test --coverage
+  variables:
+    CMS_BASE_URL: 'http://ec2-34-254-183-172.eu-west-1.compute.amazonaws.com'
```

Comment:
The production variables are prefixed with `PROD`. We could do it differently, if you prefer.
https://github.com/yldio/asap-hub/pull/69/files#diff-96edf7a6f008de9e928d04e1ae5e12a5R57

The prod endpoint isn't declared because it is the default prod one (See: https://github.com/yldio/asap-hub/pull/69/files#diff-35e6398db1c5439b9356bb5a4ea7214bR23)